code/modules/clothing/gloves/stungloves.dm

```diff
@@ -59,3 +59,28 @@
 	if(ishuman(src.loc))
 		var/mob/living/carbon/human/H = src.loc
 		H.update_inv_gloves()
+
+/obj/item/clothing/gloves/Touch(atom/A, mob/living/user, prox)
+	if(!isliving(A))
+		return
+	if(!cell)
+		return
+
+	var/mob/living/L = A
+	if(user.a_intent == I_HURT)//Stungloves. Any contact will stun the alien.
+		visible_message("<span class='danger'>\The [A] has been touched with the stun gloves by [user]!</span>")
```

Comment:
Just making sure it's what you want, but this message will show even when it's out of charge.

Not such a bad thing imo, but it should have an attempted to stunglove attack log if you do intend to keep it.